runtime/evaluate.h

```diff
@@ -34,6 +34,12 @@ class eval_context : public symbol_map<index_t> {
   std::function<void(task)> enqueue_one;
   std::function<void(std::function<bool()>)> wait_for;
 
+  // Functions implementing buffer data movement:
+  // - `copy` should copy from `src` to `dst`, filling `dst` with `padding` when out of bounds of `src`. Can be set to `slinky::copy`.
+  // - `pad` should fill the area out of bounds of `src_dims` with `padding` in `dst`. Can be set to `slinky::pad`.
+  std::function<void(const raw_buffer& src, const raw_buffer& dst, const void* padding)> copy;
```

Comment:
Shouldn't they just default-init to values of `slinky::copy/pad`?

Reply:
I suppose so. I thought about it, and none of the other callbacks did that, so I decided to be consistent. But all the others would need some wrapper, so it makes more sense in this case.